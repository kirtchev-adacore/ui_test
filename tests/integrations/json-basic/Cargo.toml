[package]
name = "json_basic"
version = "0.1.0"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
serde_derive = "1.0"

[dev-dependencies]
ui_test = { path = "../../.."}
tempfile = "3.3.0"

[[test]]
name = "ui_tests"
harness = false

# Regression check for #225
[lib]
crate-type = ["staticlib", "lib"]
