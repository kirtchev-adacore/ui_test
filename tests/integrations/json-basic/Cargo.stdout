
running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished 

{ "type": "suite", "event": "started" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/aux_attr_proc_macro.rs () - ../basic/tests/actual_tests/aux_attr_proc_macro.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/aux_derive.rs () - ../basic/tests/actual_tests/aux_derive.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/aux_proc_macro.rs () - ../basic/tests/actual_tests/aux_proc_macro.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/dep_derive.rs () - ../basic/tests/actual_tests/dep_derive.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/error_above.rs () - ../basic/tests/actual_tests/error_above.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/executable.rs () - ../basic/tests/actual_tests/executable.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/foomp-rustfix.rs () - ../basic/tests/actual_tests/foomp-rustfix.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/foomp.rs () - ../basic/tests/actual_tests/foomp.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/joined_above.rs () - ../basic/tests/actual_tests/joined_above.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/joined_below.rs () - ../basic/tests/actual_tests/joined_below.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/joined_mixed.rs () - ../basic/tests/actual_tests/joined_mixed.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/mac_span.rs () - ../basic/tests/actual_tests/mac_span.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/match_diagnostic_code.rs () - ../basic/tests/actual_tests/match_diagnostic_code.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/no_rustfix.rs () - ../basic/tests/actual_tests/no_rustfix.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/rustfix-multiple.rs () - ../basic/tests/actual_tests/rustfix-multiple.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/stdin.rs () - ../basic/tests/actual_tests/stdin.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/unicode.rs () - ../basic/tests/actual_tests/unicode.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/windows_paths.rs () - ../basic/tests/actual_tests/windows_paths.rs" }
{ "type": "test", "event": "started", "name": "../basic/tests/actual_tests/subdir/aux_proc_macro.rs () - ../basic/tests/actual_tests/subdir/aux_proc_macro.rs" }
{ "type": "test", "event": "failed", "name": "../basic/tests/actual_tests/aux_attr_proc_macro.rs (Building aux file basic/tests/actual_tests/auxiliary/proc_macro_attr.rs) - ../basic/tests/actual_tests/aux_attr_proc_macro.rs", "stdout": "command: <Building aux file basic/tests/actual_tests/auxiliary/proc_macro_attr.rs> stdout: <> stderr: <>" }
{ "type": "test", "event": "failed", "name": "../basic/tests/actual_tests/aux_attr_proc_macro.rs () - ../basic/tests/actual_tests/aux_attr_proc_macro.rs", "stdout": "command: <reading aux file `basic/tests/actual_tests/auxiliary/proc_macro_attr.rs`> stdout: <basic/tests/actual_tests/auxiliary/proc_macro_attr.rs> stderr: <>" }
{ "type": "test", "event": "failed", "name": "../basic/tests/actual_tests/aux_derive.rs (Building aux file basic/tests/actual_tests/auxiliary/derive_proc_macro.rs) - ../basic/tests/actual_tests/aux_derive.rs", "stdout": "command: <Building aux file basic/tests/actual_tests/auxiliary/derive_proc_macro.rs> stdout: <> stderr: <>" }
{ "type": "test", "event": "failed", "name": "../basic/tests/actual_tests/aux_derive.rs () - ../basic/tests/actual_tests/aux_derive.rs", "stdout": "command: <reading aux file `basic/tests/actual_tests/auxiliary/derive_proc_macro.rs`> stdout: <basic/tests/actual_tests/auxiliary/derive_proc_macro.rs> stderr: <>" }
{ "type": "test", "event": "failed", "name": "../basic/tests/actual_tests/aux_proc_macro.rs (Building aux file basic/tests/actual_tests/auxiliary/the_proc_macro.rs) - ../basic/tests/actual_tests/aux_proc_macro.rs", "stdout": "command: <Building aux file basic/tests/actual_tests/auxiliary/the_proc_macro.rs> stdout: <> stderr: <>" }
{ "type": "test", "event": "failed", "name": "../basic/tests/actual_tests/aux_proc_macro.rs () - ../basic/tests/actual_tests/aux_proc_macro.rs", "stdout": "command: <reading aux file `basic/tests/actual_tests/auxiliary/the_proc_macro.rs`> stdout: <basic/tests/actual_tests/auxiliary/the_proc_macro.rs> stderr: <>" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/dep_derive.rs (Building dependencies) - ../basic/tests/actual_tests/dep_derive.rs" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/dep_derive.rs () - ../basic/tests/actual_tests/dep_derive.rs" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/dep_derive.rs (run) - ../basic/tests/actual_tests/dep_derive.rs" }
{ "type": "test", "event": "failed", "name": "../basic/tests/actual_tests/error_above.rs () - ../basic/tests/actual_tests/error_above.rs", "stdout": "command: <\"rustc\" \"--error-format=json\" \"--out-dir\" \"$DIR/tests/integrations/json-basic/../../../target/$TMP/0/../basic/tests/actual_tests\" \"../basic/tests/actual_tests/error_above.rs\" \"--extern\" \"serde_derive=$DIR/tests/integrations/json-basic/../../../target/$TMP/0/debug/libserde_derive-5185dac99885ddee.so\" \"--extern\" \"json_basic=$DIR/tests/integrations/json-basic/../../../target/$TMP/$TRIPLE/debug/libjson_basic.a\" \"--extern\" \"json_basic=$DIR/tests/integrations/json-basic/../../../target/$TMP/$TRIPLE/debug/libjson_basic.rlib\" \"-L\" \"$DIR/tests/integrations/json-basic/../../../target/$TMP/0/debug\" \"-L\" \"$DIR/tests/integrations/json-basic/../../../target/$TMP/$TRIPLE/debug\" \"--edition\" \"2021\"> stdout: <error[E0432]: unresolved import `basic`\n --> ../basic/tests/actual_tests/error_above.rs:1:5\n  |\n1 | use basic::add;\n  |     ^^^^^ use of undeclared crate or module `basic`\n\nerror: aborting due to 1 previous error\n\nFor more information about this error, try `rustc --explain E0432`.\n> stderr: <>" }
{ "type": "test", "event": "failed", "name": "../basic/tests/actual_tests/executable.rs () - ../basic/tests/actual_tests/executable.rs", "stdout": "command: <\"rustc\" \"--error-format=json\" \"--out-dir\" \"$DIR/tests/integrations/json-basic/../../../target/$TMP/0/../basic/tests/actual_tests\" \"../basic/tests/actual_tests/executable.rs\" \"--extern\" \"serde_derive=$DIR/tests/integrations/json-basic/../../../target/$TMP/0/debug/libserde_derive-5185dac99885ddee.so\" \"--extern\" \"json_basic=$DIR/tests/integrations/json-basic/../../../target/$TMP/$TRIPLE/debug/libjson_basic.a\" \"--extern\" \"json_basic=$DIR/tests/integrations/json-basic/../../../target/$TMP/$TRIPLE/debug/libjson_basic.rlib\" \"-L\" \"$DIR/tests/integrations/json-basic/../../../target/$TMP/0/debug\" \"-L\" \"$DIR/tests/integrations/json-basic/../../../target/$TMP/$TRIPLE/debug\" \"--edition\" \"2021\"> stdout: <error[E0432]: unresolved import `basic`\n --> ../basic/tests/actual_tests/executable.rs:1:5\n  |\n1 | use basic::add;\n  |     ^^^^^ use of undeclared crate or module `basic`\n\nerror: aborting due to 1 previous error\n\nFor more information about this error, try `rustc --explain E0432`.\n> stderr: <>" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/foomp-rustfix.rs () - ../basic/tests/actual_tests/foomp-rustfix.rs" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/foomp-rustfix.rs () - ../basic/tests/actual_tests/foomp-rustfix.fixed" }
{ "type": "test", "event": "failed", "name": "../basic/tests/actual_tests/foomp.rs () - ../basic/tests/actual_tests/foomp.rs", "stdout": "command: <\"rustc\" \"--error-format=json\" \"--out-dir\" \"$DIR/tests/integrations/json-basic/../../../target/$TMP/0/../basic/tests/actual_tests\" \"../basic/tests/actual_tests/foomp.rs\" \"--extern\" \"serde_derive=$DIR/tests/integrations/json-basic/../../../target/$TMP/0/debug/libserde_derive-5185dac99885ddee.so\" \"--extern\" \"json_basic=$DIR/tests/integrations/json-basic/../../../target/$TMP/$TRIPLE/debug/libjson_basic.a\" \"--extern\" \"json_basic=$DIR/tests/integrations/json-basic/../../../target/$TMP/$TRIPLE/debug/libjson_basic.rlib\" \"-L\" \"$DIR/tests/integrations/json-basic/../../../target/$TMP/0/debug\" \"-L\" \"$DIR/tests/integrations/json-basic/../../../target/$TMP/$TRIPLE/debug\" \"--edition\" \"2021\"> stdout: <error[E0432]: unresolved import `basic`\n --> ../basic/tests/actual_tests/foomp.rs:1:5\n  |\n1 | use basic::add;\n  |     ^^^^^ use of undeclared crate or module `basic`\n\nerror: aborting due to 1 previous error\n\nFor more information about this error, try `rustc --explain E0432`.\n> stderr: <>" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/joined_above.rs () - ../basic/tests/actual_tests/joined_above.rs" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/joined_above.rs () - ../basic/tests/actual_tests/joined_above.fixed" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/joined_below.rs () - ../basic/tests/actual_tests/joined_below.rs" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/joined_below.rs () - ../basic/tests/actual_tests/joined_below.fixed" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/joined_mixed.rs () - ../basic/tests/actual_tests/joined_mixed.rs" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/joined_mixed.rs () - ../basic/tests/actual_tests/joined_mixed.fixed" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/mac_span.rs () - ../basic/tests/actual_tests/mac_span.rs" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/mac_span.rs () - ../basic/tests/actual_tests/mac_span.fixed" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/match_diagnostic_code.rs () - ../basic/tests/actual_tests/match_diagnostic_code.rs" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/match_diagnostic_code.rs () - ../basic/tests/actual_tests/match_diagnostic_code.fixed" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/no_rustfix.rs () - ../basic/tests/actual_tests/no_rustfix.rs" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/rustfix-multiple.rs () - ../basic/tests/actual_tests/rustfix-multiple.rs" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/rustfix-multiple.rs () - ../basic/tests/actual_tests/rustfix-multiple.1.fixed" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/rustfix-multiple.rs () - ../basic/tests/actual_tests/rustfix-multiple.2.fixed" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/stdin.rs () - ../basic/tests/actual_tests/stdin.rs" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/stdin.rs (run) - ../basic/tests/actual_tests/stdin.rs" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/unicode.rs () - ../basic/tests/actual_tests/unicode.rs" }
{ "type": "test", "event": "ok", "name": "../basic/tests/actual_tests/windows_paths.rs () - ../basic/tests/actual_tests/windows_paths.rs" }
{ "type": "test", "event": "failed", "name": "../basic/tests/actual_tests/subdir/aux_proc_macro.rs () - ../basic/tests/actual_tests/subdir/aux_proc_macro.rs", "stdout": "command: <\"Building aux file basic/tests/actual_tests/auxiliary/the_proc_macro.rs\"> stdout: <previous build failed> stderr: <>" }
{ "type": "suite", "event": "failed", "passed": 22, "failed": 7, "ignored": 0, "measured": 0, "filtered_out": 0 }

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished 

